formatter/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "formatter"
+version = "0.17.0"
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+description = "A formatter for Miden Assembly (.masm) files"
+keywords = ["miden", "zkVM", "assembly", "masm"]
+categories = ["command-line-utilities", "development-tools"]
+
+[dependencies]
+glob = "0.3"
+clap = { version = "4.1", features = ["derive"] }
+regex = "1"
+once_cell = "1.20.3"
+tempfile = "3.17.1"
+
+[[bin]]
+name = "masm-fmt"
+path = "src/main.rs"
```

Comment:
I think it would be good to add installation instructions (from crates.io if published and the local one) in the Readme or somewhere appropriate.